src/com/amazon/corretto/crypto/provider/RsaGen.java

```diff
@@ -3,23 +3,17 @@
 
 package com.amazon.corretto.crypto.provider;
 
-import java.math.BigInteger;
 import java.security.InvalidAlgorithmParameterException;
 import java.security.InvalidParameterException;
 import java.security.KeyFactory;
 import java.security.KeyPair;
 import java.security.KeyPairGeneratorSpi;
-import java.security.NoSuchAlgorithmException;
-import java.security.PrivateKey;
-import java.security.PublicKey;
 import java.security.SecureRandom;
 import java.security.spec.AlgorithmParameterSpec;
-import java.security.spec.InvalidKeySpecException;
 import java.security.spec.RSAKeyGenParameterSpec;
-import java.security.spec.RSAPrivateCrtKeySpec;
-import java.security.spec.RSAPublicKeySpec;
 
 class RsaGen extends KeyPairGeneratorSpi {
+    private static final RSAKeyGenParameterSpec DEFAULT_KEYGEN_SPEC = new RSAKeyGenParameterSpec(2048, RSAKeyGenParameterSpec.F4);
```

Comment:
yeah, that would make sense.

[poking around google a bit](https://www.yubico.com/blog/big-debate-2048-4096-yubicos-stand/), consensus seems to be that 2048 is a reasonable default for the next decade-ish as increase in bits of security doesn't really scale well with key length, so just hard-coding 2048 seems like it would be fine too.